### PR TITLE
Improvement/add services to wdio

### PR DESCRIPTION
**Pull request by qa-joerg**
Increase timeout to 60000 and add w3c false to WdioConfigTemplate.ts. Also add 'devtools', 'image-comparison' to ConfigTemplate.ts to use this feature

### CODE REVIEW

File: src/commands/e2e/WdioConfigTemplate.ts [hunk 1]
```diff
@@ -157,6 +158,7 @@ exports.config = {
                 browserName: '${this.browser}',
                 'moz:firefoxOptions': {
                     'args': ['-headless']
+                    w3c: false,
```
Comment (stefancplace) [inline src/commands/e2e/WdioConfigTemplate.ts:161]: Is Internet Explorer affected as well?

**Comment by stefancplace** [comment]
Nice that you are taking care of that 👍

File: src/commands/e2e/WdioConfigTemplate.ts (hunk 1 above)
Comment (qa-joerg) [reply src/commands/e2e/WdioConfigTemplate.ts:161]: yes all browser are affected by this. This change works for chrome but not for the other browser

Comment (stefancplace) [reply src/commands/e2e/WdioConfigTemplate.ts:161]: do we need it then for firefox?

Comment (qa-joerg) [reply src/commands/e2e/WdioConfigTemplate.ts:161]: No, I need to update this again or find another solution for this

File: src/commands/e2e/WdioConfigTemplate.ts [hunk 2]
```diff
@@ -143,6 +143,7 @@ exports.config = {
                 maxInstances: 1,
                 browserName: '${this.browser}',
                 'goog:chromeOptions': {
+                    w3c: false,
```
Comment (stefancplace) [inline src/commands/e2e/WdioConfigTemplate.ts:146]: do you only need w3c in headless ?
probably we need to introduce w3c option for chrome in normal mode as well?
I will add it hope that`s fine

Comment (qa-joerg) [reply src/commands/e2e/WdioConfigTemplate.ts:146]: w3c is needed in any case

**Review by stefancplace — APPROVED** [review]
LGTM, maybe you could check locally @qa-joerg to verify configs are valid

**Pull request merged**